app/assets/stylesheets/user_groups/_index.css.scss

```diff
@@ -9,46 +9,46 @@
       position: relative;
       box-shadow: 0px 0px 1px #B7CDBF;
       text-align: center;
-      padding: 3rem 0 0 0 !important;
       width: 80%;
       margin: 0 auto;
       background: white;
       height: 17rem;
       border-radius: 0.4rem;
 
-      &:hover {
-        i.fa { color: $green; }
-      }
 
       .default-group {
         position: absolute;
         top: 0;
         right: 0;
-        padding: 3px;
         border-top-right-radius: 0.4rem;
         border-top-left-radius: 0.4rem;
         background: #E7E7E7;
-        box-shadow: 0px 0px 1px rgb(216, 216, 216);
         cursor: pointer;
-        height: 1.25rem;
+        height: 1.3rem;
         width: 100%;
+        line-height: 1.3;
 
         .fa-check {
           position: absolute;
-          top: 3px;
-          right: 6px;
+          top: 2px;
+          right: 5px;
           color: $bright-green;
           display: none;
         }
 
         &:hover {
+          background: $bright-green;
+          color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.